src/Microsoft.DotNet.XHarness.CLI/Program.cs

```diff
@@ -28,7 +30,8 @@ public static int Main(string[] args)
 
         if (shouldOutput)
         {
-            Console.WriteLine($"XHarness command issued: {string.Join(' ', args)}");
+            var version = FileVersionInfo.GetVersionInfo(Assembly.GetExecutingAssembly().Location).ProductVersion;

```

Comment:
Since we are getting this string in the --version argument as well, can we refactor this call to a function so the logic is at one place?

Reply:
Good point, done